src/main/assets/js/phone-numbers.ts

```diff
@@ -0,0 +1,123 @@
+import $ from 'jquery';
```

Comment:
It might be worth renaming this to 'contacts' in future to keep consistency with the controller. Same with the tests related to this new tab.

Reply:
Yeah - we could rename in future. The back-end is 'contacts' (FaCT API) but the front-end was to display as 'phone numbers', which is why the client-side files are named as phone-numbers and the controller named contacts.